extensions/amp-social-share/0.2/social-share.js

```diff
@@ -83,6 +83,7 @@ export function SocialShare(props) {
       onClick={handleActivation}
       style={{...size, ...props['style']}}
       {...props}
+      aria-label={`amp-social-share component of type: ${type}`}
```

Comment:
We should probably make this a bit more user-friendly along the lines of 'Share on ${type}' ?

Reply:
Yeah, certainly. It's a user-readable string.